src/zenml/orchestrators/step_runner.py

```diff
@@ -354,12 +367,19 @@ def _store_output_artifacts(
         assert artifact_stores is not None  # Every stack has an artifact store.
         artifact_store_id = artifact_stores[0].id
         output_artifacts: Dict[str, ArtifactRequestModel] = {}
+        output_artifact_metadata: Dict[str, Dict[str, "MetadataType"]] = {}
         for output_name, return_value in output_data.items():
             materializer_class = output_materializers[output_name]
             materializer_source = self.configuration.outputs[
                 output_name
             ].materializer_source
             uri = output_artifact_uris[output_name]
+            materializer = materializer_class(uri)
+            materializer.save(return_value)
+            # TODO: or self._pipeline.enable_artifact_metadata
```

Comment:
todo

Reply:
almost forgot, thanks for reminding 🙏

Reply:
Will include in the upcoming `enable_artifact_metadata` rework

Reply:
Fixed